lib/cocoapods/generator/module_map.rb

```diff
@@ -90,12 +90,9 @@ def module_specifier_prefix
       end
 
       # The suffix attributes to `module`.
-      # Ensures that library module maps are treated as `system` modules,
-      # supressing warnings when imported, as is done for header imports given via `-isystem`.
       #
       def module_declaration_attributes
-        return '' if target.requires_frameworks?
-        ' [system]'
+        ''
```

Comment:
kept the method in case we need it for some other attributes to include in the future.